data/railsconf/railsconf-2023/videos.yml

```
@@ -0,0 +1,1044 @@
+  raw_title: RailsConf 2023 - Functional Patterns in Ruby by John Crepezzi
+  speakers:
+    - John Crepezzi
+  description: I recently started working primarily in a statically-typed functional
+    programming language (OCaml). While learning, I spent a lot of time trying to
+    fit OCaml into a Ruby-shaped box. While there are plenty of things that I miss
+    about Ruby day-to-day, there are also a lot of good lessons to take away! No,
+    I’m not talking about static typing, or some rant on how nil is an anti-pattern.
+    This talk instead will dig into concepts from around the functional world and
+    how they can be modeled in Ruby for cleaner, more future-proof code.
+  thumbnail_xs: https://i.ytimg.com/vi/3ndcqh9fTGA/default.jpg
+  thumbnail_sm: https://i.ytimg.com/vi/3ndcqh9fTGA/mqdefault.jpg
+  thumbnail_md: https://i.ytimg.com/vi/3ndcqh9fTGA/hqdefault.jpg
+  thumbnail_lg: https://i.ytimg.com/vi/3ndcqh9fTGA/sddefault.jpg
+  thumbnail_xl: https://i.ytimg.com/vi/3ndcqh9fTGA/maxresdefault.jpg
+  published_at: '2023-07-10'
+  video_id: 3ndcqh9fTGA
+- title: Who Wants to be a Ruby Engineer?
+  raw_title: RailsConf 2023 - Who Wants to be a Ruby Engineer? by Drew Bragg
+  speakers:
+    - Drew Bragg
+  description: 'Welcome to the Ruby game show where contestants try to guess the output
+    of a small bit of Ruby code. Sound easy? Here''s the challenge: the snippets come
+    from some of the weirdest parts of the Ruby language.  The questions aren''t easy,
+    but et enough right to be crowned a "Strange" Ruby Engineer and win a fabulous
+    prize.'
+  thumbnail_xs: https://i.ytimg.com/vi/4VT0aDEDd28/default.jpg
+  thumbnail_sm: https://i.ytimg.com/vi/4VT0aDEDd28/mqdefault.jpg
+  thumbnail_md: https://i.ytimg.com/vi/4VT0aDEDd28/hqdefault.jpg
+  thumbnail_lg: https://i.ytimg.com/vi/4VT0aDEDd28/sddefault.jpg
+  thumbnail_xl: https://i.ytimg.com/vi/4VT0aDEDd28/maxresdefault.jpg
+  published_at: '2023-07-10'
+  video_id: 4VT0aDEDd28
+- title: Building an offline experience with a Rails-powered PWA
+  raw_title: RailsConf 2023 - Building an offline experience with a Rails-powered
+    PWA by Alicia Rojas
+  speakers:
+    - Alicia Rojas
+  description: Progressive web applications (PWAs) allow us to provide rich offline
+    experiences as native apps would, while still being easy to use and share, as
+    web apps are. Come to learn how to turn your regular Rails application into a
+    PWA, taking advantage of the new front-end tools that come with Rails by default
+    since version 7. Entry-level developers are welcome!
+  thumbnail_xs: https://i.ytimg.com/vi/4cQuxQdWkPE/default.jpg
+  thumbnail_sm: https://i.ytimg.com/vi/4cQuxQdWkPE/mqdefault.jpg
+  thumbnail_md: https://i.ytimg.com/vi/4cQuxQdWkPE/hqdefault.jpg
+  thumbnail_lg: https://i.ytimg.com/vi/4cQuxQdWkPE/sddefault.jpg
+  thumbnail_xl: https://i.ytimg.com/vi/4cQuxQdWkPE/maxresdefault.jpg
+  published_at: '2023-07-10'
+  video_id: 4cQuxQdWkPE
+- title: Accessible
```

Looks like we are missing half of the title here
```suggestion
- title: Accessible by default
```